druid/src/widget/widget.rs

```diff
@@ -222,6 +222,46 @@ pub trait Widget<T> {
             ..Default::default()
         }
     }
+
+    /// Max intrinsic/preferred width is the width a widget would take given infinite width constraint.
```

Comment:
This is a little different from how intrinsic width is [defined in Flutter](https://api.flutter.dev/flutter/rendering/RenderFractionallySizedOverflowBox/computeMaxIntrinsicWidth.html), where max intrinsic width is *"the smallest width beyond which increasing the width never decreases the preferred height"*. Is the difference intentional, and if so could you elaborate on why?

Reply:
I honestly feel it makes the definition a little convoluted. For its intents and purposes, what I have is an easier version of that.